src/sidebar/components/Annotation/AnnotationPublishControl.js

```diff
@@ -85,7 +85,6 @@ function AnnotationPublishControl({
           style={buttonStyle}
           onClick={onSave}
           disabled={isDisabled}
-          title={`Publish this annotation to ${publishDestination}`}
```

Comment:
I think the origin of the problem here was that the `title` attribute was intended to set the tooltip originally, not the accessible label (`aria-label`). In that situation having a tooltip which was more verbose than the button label was OK, eg. to provide an extended description of the action that couldn't fit in the button text. All of the `*Button` components in `@hypothesis/frontend-shared` currently always set `aria-label` to match `title`. 

In this particular scenario, the tooltip doesn't provide much additional value over the button text IMO so I think it's OK to just drop it. I wonder however if there are any scenarios where an extended tooltip does provide significant value over the button text and which we'll want to expose some other way.

I think we might have had this discussion in the past, but if we have I've forgotten the conclusion. CC-ing @lyzadanger in case she knows more.

Reply:
Yep, I had this thought too, lets wait to see what Lyza thinks. Because as the spec is written it sorta means that title can't be used with button text--or thats how I at least read it.

More details here:

https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html